server/templates/place.html

```diff
@@ -32,6 +40,8 @@
 {% elif category != '' %}
 <link rel="canonical" href="https://{{ request.host }}{{ canonical_category_url }}" />
 {% endif %}
+
+<meta name="description" content="{{ description }}" />
```

Comment:
Can you use the `place_summary` here if it's available, otherwise default to the description

Reply:
Great idea, done!